src/components/error/error.component.jsx

```diff
@@ -4,7 +4,7 @@ const Error = ({title, summary, link1, link2}) => {
             className="govuk-error-summary optional-extra-class lbh-error-summary"
             aria-labelledby="error-summary-title"
             role="alert"
-            tabIndex="-1"
+            tabindex="-1"
```

Comment:
```suggestion
            tabIndex="-1"
```
Not to be picky, it just causes a warning when making a build, which might make the pipeline fail :)